src/application/tasks/ventilation.c

```diff
@@ -6,12 +6,12 @@
 
 #include <stdbool.h>
 #include <stdint.h>
-
-#include "platform/system_info.h"
-#include "platform/sensor.h"
-#include "platform/valve.h"
-#include "application/tasks/ventilation.h"
-#include "application/dss.h"
+#include <platform/system_info.h>
```

Comment:
attention all the includes should use "" format since the  files you include is our own and not the system ones